Organic/Organic.php

```diff
@@ -487,18 +482,35 @@ public function getCategoryForCurrentPage() {
         }
     }
 
+    public function getSlugs( ...$terms ) {
+        return array_map(
+            fn( $term ) => $term->slug,
```

Comment:
https://www.php.net/manual/en/functions.arrow.php
> Arrow functions were introduced in PHP 7.4 as a more concise syntax for anonymous functions.

But I believe we should support PHP 7.0
https://github.com/orgnc/wordpress-plugin/blob/master/phpcs.xml#L59

@jdemaris, could you confirm?
Is the PHP 7.0 version the minimum requirement?
If yes, why this PR mentions PHP 5?
https://github.com/orgnc/wordpress-plugin/pull/72

Reply:
Fixed by using old syntax.